### ISSUE TITLE
sdwan_switchport_feature_template expects "static_mac_addresses" always

### ISSUE THREAD

**Issue by rrahimm** [CiscoDevNet/terraform-provider-sdwan#148]
When we create a switchport feature template using the provider, the GUI is never able to show up the configured values.

![image](https://github.com/CiscoDevNet/terraform-provider-sdwan/assets/39684933/ad9c54f4-13bb-45c5-ab95-9166c6e25c08)

on further evaluation, it was identified that the optional variable "static_mac_addresses" was always needed to have the template properly get configured and viewable in GUI.

Working TF Plan
```yaml
# module.sdwan.sdwan_switchport_feature_template.switchport_feature_template["FT-CEDGE-SWITCHPORT-V01"] will be created
  + resource "sdwan_switchport_feature_template" "switchport_feature_template" {
      + age_out_time         = 500
      + description          = "Switchport Template"
      + device_types         = [
          + "vedge-C8000V",
        ]
      + id                   = (known after apply)
      + interfaces           = [
          + {
              + duplex                         = "full"
              + name                           = "GigabitEthernet0/1/0"
              + optional                       = false
              + shutdown                       = false
              + speed                          = "1000"
              + switchport_access_vlan         = 10
              + switchport_mode                = "access"
              + switchport_trunk_allowed_vlans = "4 5 10-20"
              + switchport_trunk_native_vlan   = 100
            },
        ]
      + module_type          = "8"
      + name                 = "FT-CEDGE-SWITCHPORT-V01"
      + slot                 = 0
      + static_mac_addresses = [
          + {
              + if_name     = "GigabitEthernet0/0/0"
              + mac_address = "0000.0000.0000"
              + vlan        = 1000
            },
        ]
      + sub_slot             = 1
      + template_type        = (known after apply)
      + version              = (known after apply)
    } 
```
    
non-working TF Plan
```yaml
# module.sdwan.sdwan_switchport_feature_template.switchport_feature_template["FT-CEDGE-SWITCHPORT-V01"] will be created
  + resource "sdwan_switchport_feature_template" "switchport_feature_template" {
      + age_out_time         = 500
      + description          = "Switchport Template"
      + device_types         = [
          + "vedge-C8000V",
        ]
      + id                   = (known after apply)
      + interfaces           = [
          + {
              + duplex                         = "full"
              + name                           = "GigabitEthernet0/1/0"
              + optional                       = false
              + shutdown                       = false
              + speed                          = "1000"
              + switchport_access_vlan         = 10
              + switchport_mode                = "access"
              + switchport_trunk_allowed_vlans = "4 5 10-20"
              + switchport_trunk_native_vlan   = 100
            },
        ]
      + module_type          = "8"
      + name                 = "FT-CEDGE-SWITCHPORT-V01"
      + slot                 = 0
      + static_mac_addresses = []
      + sub_slot             = 1
      + template_type        = (known after apply)
      + version              = (known after apply)
    }
 ```

**Comment by danischm** [comment]
https://github.com/CiscoDevNet/terraform-provider-sdwan/commit/6bd67153429ed035d3548017afc313195267f830

**Comment by danischm** [comment]
Fixed in v0.3.5 release.

**Comment by rrahimm** [comment]
tested
